media/js/base/mozilla-notification-banner.js

```diff
@@ -16,6 +16,7 @@ if (typeof Mozilla === 'undefined') {
     NotificationBanner.COOKIE_CODE_ID = 'mozilla-notification-banner';
     NotificationBanner.COOKIE_EXPIRATION_DAYS = 21; // default cookie expiry 21 days
     NotificationBanner.COOKIE_INTERACTION_VALUE = 'interacted';
+    NotificationBanner.SAMPLE_RATE = 0.05; // 5% sample rate limit
```

Comment:
Ok, this can't be per config, since we need to know the sample rate before a config is selected. What i can do it make a couple of convenience methods to get/set a sample rate which can be configured per instance.